py4DSTEM/datacube/datacube.py

```diff
@@ -1135,7 +1103,7 @@ def get_beamstop_mask(
         )
 
         # Add to tree
-        self.tree(x)
+        self.attach( mask_beamstop )
```

Comment:
oh dear!!  This code has been broken and fixed and broken again over and over.  First time I've seen this permutation though.  Very exciting.  It needs to be `self.attach( x )`